HACKING.md

```diff
@@ -169,7 +169,23 @@ EOF
 Then create the secret in OpenShift:
 
 ```
-oc create secret generic aws-fcos-builds-bot-config --from-file=config=/path/to/file
+oc create secret generic aws-fcos-builds-bot-config --from-file=config=/path/to/upload-secret
+```
+
+We also have a second AWS config that can be used for running kola
+tests. If you have a single account that has enough permissions for
+both then you can use the same account for both uploading builds and
+running kola tests (i.e. re-use `upload-secret` from above. If not then
+you can use a second set of credentials for the kola tests.
+
+```
+cat <<'EOF' > /path/to/kola-secret
+[default]
+aws_access_key_id=keyid
+aws_secret_access_key=key
+EOF
+
+oc create secret generic aws-fcos-kola-bot-config --from-file=config=/path/to/kola-secret
```

Comment:
Is the IAM account called `fcos-kola-bot`? I'd like to keep the exact name of the IAM in the secret name to make things easier to track.